modules/nextflow/src/test/groovy/nextflow/SessionTest.groovy

```diff
@@ -601,16 +601,22 @@ class SessionTest extends Specification {
 
     }
 
+    @Unroll
     def 'should get module binaries status'() {
         given:
-        def session = new Session(CONFIG)
+        def session = new Session()
+        NextflowMeta.instance.moduleBinaries(MODE)
 
         expect:
         session.enableModuleBinaries() == EXPECTED
-        
+
+        cleanup:
+        NextflowMeta.instance.moduleBinaries(false)
+
         where:
-        CONFIG                                      | EXPECTED
-        [:]                                         | false
-        [nextflow:[enable:[moduleBinaries: true]]]  | true
+        MODE  | EXPECTED
+        false | false
+        true  | true
+
```

Comment:
is config setting still allowed right? then test with the config should stay

Reply:
With the changes of the PR enableModuleBinaries is got from NextflowMeta, not navigating the sesion.config as before. So, it is not possible to test the functionality as before at session level. The functionality is tested in CmdRunTest, where we test we get the correct moduleBinaries flag in config and script are set to NextflowMeta. In this test, we just check session is getting the correct value from NextflowMeta.

Reply:
umm? is still possible to set this option via the config?

Reply:
Yes, I followed the same procedure as for the nextflow.enable.strict in [CmdRun](https://github.com/nextflow-io/nextflow/blob/58b6961e1f7edaf63c6cc506e0403f72d9554a3a/modules/nextflow/src/main/groovy/nextflow/cli/CmdRun.groovy#L436C5-L442C6)